### PR TITLE
Added support for symfony 5

### PR DESCRIPTION
**Pull request by core23**
## Subject

I am targeting this branch, because this is BC.

## Changelog

```markdown
### Added
- Added support for symfony 5

### Removed
- Dropped support for PHP 7.1
```

### CODE REVIEW

**Comment by core23** [comment]
We need PHPUnit 8 to make this pass :/

**Comment by franmomu** [comment]
> We need PHPUnit 8 to make this pass :/

yep, in [twig-extensions](https://github.com/sonata-project/dev-kit/pull/522) was the same thing

**Comment by OskarStark** [comment]
But it looks like Travis is still red with PHPUnit 8

File: composer.json [hunk 1]
```diff
@@ -22,23 +22,23 @@
...
+        "symfony/intl": "^3.4 || ^4.2 || ^5.0",
+        "symfony/templating": "^3.4 || ^4.2 || ^5.0",
         "twig/twig": "^2.9"
     },
     "conflict": {
         "sonata-project/user-bundle": "<2.0 || >=5.0"
     },
     "require-dev": {
-        "matthiasnoback/symfony-dependency-injection-test": "^3.1",
+        "matthiasnoback/symfony-dependency-injection-test": "^3.1 || ^4.0",
```
Comment (franmomu) [inline composer.json]:
```suggestion
        "matthiasnoback/symfony-dependency-injection-test": "^4.0",
```
Using PHPUnit 8 it should be 4.0

**Comment by core23** [comment]
Closing in favor of  #280

**Pull request closed without merging**